Data/Scripts/RealisticThrusters/GridLogic.cs

```diff
@@ -163,6 +165,7 @@ public void Update()
             }
         }
 
+        // Note: this event doesn't seem to fire anymore, nor is it listed in the Mod API docs
```

Comment:
This is annoying, and means you need to toggle ownership (eg. Share with Friend -> Share with All) or add/remove a cockpit to trigger an update after changing CustomData.